src/transformers/modeling_bert.py

```diff
@@ -371,15 +376,34 @@ def forward(self, hidden_states, input_tensor):
         return hidden_states
 
 
+class ChunkFeedForward(nn.Module):
```

Comment:
please rename to `ChunkBertFeedForward`

Reply:
Actually, as you said this won't be possible because the layer weights would change, so we have to keep `self.intermediate` and `self.output` sadly...